build/templates/_converters.py.mako

```diff
@@ -124,6 +124,22 @@ def convert_repeated_capabilities(repeated_capability, prefix=''):
     return [prefix + r for r in _convert_repeated_capabilities(repeated_capability, prefix)]
 
 
+def convert_repeated_capabilities_from_init(repeated_capability, encoding):
```

Comment:
This function should not need to know where it's called from.
I'd rather give it a more generic name or get rid of it and join from where it's called.

`convert_repeated_capabilities_from_init`  -> `convert_repeated_capabilities_to_comma_delimited_string`
and
`convert_repeated_capabilities` -> `convert_repeated_capabilities_to_list`